assets/scss/styles.scss

```diff
@@ -14,11 +14,11 @@
 ///
 /// -> Utilities
 /// -
+@import "core/variables";
 @import "core/colors";
 @import "core/reset";
 @import "core/settings";
 @import "core/typography";
-@import "core/variables";
```

Comment:
Is this change on purpose?

Reply:
change the order, you need to load the variables first to be able to use them

Reply:
If we follow our framework pattern, variables should be included in the settings file.

Reply:
> If we follow our framework pattern, variables should be included in the settings file.

Si, tenemos que refactorizar las variables

Reply:
Creo que @damianmuti simplemente propone mover la definición de las variables a otro archivo, creo que es un cambio simple.

Si no está claro, lo podemos dejar así y luego @damianmuti puede hacer un PR rápido para corregirlo.